File: shivam-chaurasiya/Practice Problems/Level 1/01 Beginner Programs/Problem19.py
# Simple Interest with Total Amount

p = int(input('Enter the principal amout: '))
r = int(input('Enter the rate of interest: '))
t = int(input('Enter the time period: '))

simple_interest = (p*r*t)/100
total_amout = p+simple_interest

print('Simple Interest:', simple_interest)
print('Total Amount:', total_amout)

File: Basic Programs/Problem_19.js
/*--------------------------------
Problem 19: Simple Interest with Total Amount

Write a program that takes principal amount, rate of interest, and time period as input, calculates simple
interest and total amount (Principal + SI), then displays both values.
--------------------------------*/

const p = 5000;
const r = 2;
const t = 3;

const si = (p * r * t) / 100;

const amt = p + si;

console.log(`The Simple Interest is: ${si}`);
console.log(`The amount will be: ${amt}`);

/* OUTPUT:
The Simple Interest is: 300
The amount will be: 5300
*/
